docs/src/modules/components/AppNavDrawer.js

```diff
@@ -367,7 +349,7 @@ const StyledDrawer = styled(Drawer)(({ theme }) => {
 
 const SwipeableDrawerPaperComponent = styled('div')(({ theme }) => {
   return {
-    width: 280,
+    width: 292,
```

Comment:
Just as a heads-up, we need to change the width on the `AppLayoutDocs` file too, to match the change done in the NavDrawer.